### ISSUE TITLE
Urls to DDR objects on Source detail pages are malformed

### ISSUE THREAD

**Issue by GeoffFroh** [denshoproject/encyc-front#90]
Urls pointing at objects from the DDR on Source detail pages are malformed. See, for example:

https://encycstage.densho.org/sources/en-ddr-densho-266-32-23-1/

DDR ID is malformed in url between the image and the caption:

http://ddr.densho.org/ddr/densho/266/32/ should be http://ddr.densho.org/ddr-densho-266-32/

**Comment by gjost** [comment]
The `external_url` field in Elasticsearch is malformed:
```
    external_url: "http://ddr.densho.org/ddr/densho/266/32/",
```
Another issue with `encyc-core`.

**Comment by gjost** [comment]
Many Source records in the PSMS (the ones that point to DDR) have the URL formatted this way.  I think this is a previous DDR URL format. Note: there are many records which point elsewhere e.g. log.gov.
I'll write a regex in `encyc-core` that reformats the DDR URLs.

**Comment by gjost** [comment]
Fixed in `encyc-core` commit `a8fb961` for package `encyccore-master_4.0.4~deb9`.

**Comment by pkikawa** [comment]
fixed.
